README.md

```diff
@@ -1,2 +1,55 @@
+[![Build Status](https://travis-ci.org/onego-project/xmlrpc.svg?branch=master)](https://travis-ci.org/onego-project/xmlrpc)
+
 # xmlrpc
-XML-RPC Go library
+XMLRPC Go library is designed to make it easy to create a XMLRPC server that
```

Comment:
So not true :smile:. This is a client library. It's in no way useful for server creation.